### PR TITLE
Added conditional statement for Cygwin environment

### LINKED ISSUE #1906: Update tribler.sh so Tribler can run from a Cygwin environment
Currently, when running `tribler.sh` in a Cygwin environment, it will just terminate. We can easily change it so it will be able to run in Cygwin.

### PR DESCRIPTION
**Pull request by devos50**
Fixes #1906

### CODE REVIEW

File: tribler.sh [hunk 1]
```diff
@@ -27,7 +27,8 @@ if [ "$UNAME" = "Linux" ]; then
     }
 
     python2.7 $TRIBLER_SCRIPT
-
+elif [[ "$UNAME" = CYGWIN_NT* ]]; then
+    python $TRIBLER_SCRIPT
```
Comment (whirm) [inline tribler.sh]: That's not working for me.

I tested with:
```
[ HELLO = HEL* ] && echo "lel"
[ HELLO = "HEL*" ] && echo "lel"
[[ HELLO = "HEL*" ]] && echo "lel"
```
I would go for something like: ```[ ! -z `uname -s|grep ^Linux` ] && echo lel```

Comment (devos50) [inline tribler.sh]: @whirm in the 3rd example you're checking against the string "HEL*" I think. Removing the quotes around `HEL*` works for me.

Comment (devos50) [inline tribler.sh]: However, that double square bracket thingy does not work with all versions of bash, I will rewrite it to use `grep` which is more universal.

Comment (whirm) [inline tribler.sh]: None of them work for me, I just was trying random stuff, never seen that syntax before either.

**Comment by devos50** [comment]
@whirm I've rewritten the condition to use `grep`.

File: tribler.sh [hunk 2]
```diff
@@ -27,7 +27,8 @@ if [ "$UNAME" = "Linux" ]; then
     }
 
     python2.7 $TRIBLER_SCRIPT
-
+elif [ ! -z `uname -s | grep CYGWIN_NT*` ]; then
```
Comment (whirm) [inline tribler.sh]: Remove the *, that means any amount of T's at the end (or will expand to a file name if there's any matching that name on $PWD.

**Comment by devos50** [comment]
@whirm hmm to much used to use `*` as wildcard. I removed the `*`.

**Pull request merged**